tasks/src/text-generation/about.md

```diff
@@ -95,3 +95,7 @@ Would you like to learn more about the topic? Awesome! Here you can find some cu
 - [Training a CLM in PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch/language-modeling)
 - [Training a CLM in TensorFlow](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/language-modeling)
 - [Text Generation in PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch/text-generation)
+
+### Documentation
+
+- [Causal language modeling task guide](https://huggingface.co/docs/transformers/tasks/language_modeling)
```

Comment:
this page also covers text to text generation so if we have a dedicated guide feel free to add it :')

Reply:
Good thinking! I'll add one more link!